src/index.js

```diff
@@ -0,0 +1,42 @@
+import './style.css';
+
+const list = [
+  {
+    description: 'watching tutorials',
+    completed: false,
+    index: 0,
+  },
+
+  {
+    description: 'wash the dishes',
+    completed: false,
+    index: 1,
+  },
+
+  {
+    description: 'buy groceries',
+    completed: false,
+    index: 2,
+  },
+
+  {
+    description: 'complete the To Do list project',
+    completed: false,
+    index: 3,
+  },
+];
+
+const loadList = () => {
+  const listCont = document.querySelector('.list-cont');
+
```

Comment:
- [ ] Kindly sort the items list based on their index value before you display it to achieve this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=On%20page%20load%20render%20the%20dynamically%20created%20list%20of%20tasks%20in%20the%20dedicated%20placeholder.%20The%20list%20should%20appear%20in%20order%20of%20the%20index%20values%20for%20each%20task.).